t/compile.t

```diff
@@ -18,11 +18,8 @@ map {
     }, catfile($thisdir, $_));
 } qw(../bin ../lib);
 
-for (@tests) {
-    my $output = `perl -I "$libpath" -c $_ 2>&1`;
-    my $ok = ($? >> 8 == 0);
-    print $output unless $ok;
-    ok($ok, "$_ syntax check");
+foreach my $file (@tests) {
+    require_ok $file;
```

Comment:
That's odd.  Why wouldn't require_ok be using your Perl path?  I did notice $libpath, but that's needed because you're launching a new subprocess.  require_ok runs in process and should have access to the same INC that t/compile.t has access to.  Anyway, don't want to get sidetracked...

Ah, right, requires_ok isn't going to work with scripts because it will try to execute them.  I'll rebase so the test picks up the scripts, too.

What do you think about just switching to [Test::Compile](http://search.cpan.org/~egiles/Test-Compile/lib/Test/Compile.pm)?  Seems like the easiest thing to do...